packages/opera.rb

```diff
@@ -12,6 +12,7 @@ class Opera < Package
     depends_on 'alien' => :build
     depends_on 'gtk3'
     depends_on 'sommelier'
+    depends_on 'gsettings_desktop_schemas'
```

Comment:
Please add this dependency above sommelier.  Since sommelier has post install instructions, if you don't install it last, the messages will scroll past the screen and the user will miss those instructions.